addons/website/models/website.py

```diff
@@ -1526,9 +1534,13 @@ def _search_get_details(self, search_type, order, options):
 
         :return: list of search details obtained from the `website.searchable.mixin`'s `_search_get_detail()`
         """
-        result = []
-        if search_type in ['pages', 'all']:
-            result.append(self.env['website.page']._search_get_detail(self, order, options))
+        if search_type == "all":
```

Comment:
I think we use usually single quotes for technical identifier => `'all'` (and double quotes for text messages)